edi_oca/tests/test_backend_jobs.py

```diff
@@ -64,6 +65,38 @@ def test_output_fail_retry(self):
             mocked.side_effect = ReqConnectionError("Connection broken")
             with self.assertRaises(RetryableJobError):
                 job.perform()
+        self.assertEqual(record.edi_exchange_state, "output_pending")
+
+    def test_output_fail_too_many_retries(self):
+        job_counter = self.job_counter()
+        vals = {
+            "model": self.partner._name,
+            "res_id": self.partner.id,
+            "edi_exchange_state": "output_pending",
+        }
+        record = self.backend.create_record("test_csv_output", vals)
+        record._write({"create_date": "2024-01-10 09:00:00"})
+        record._set_file_content("ABC")
+        with mock.patch.object(type(self.backend), "_exchange_send") as mocked:
+            mocked.side_effect = ReqConnectionError("Connection broken")
+            with freeze_time("2024-01-10 11:00:00"):
+                # + 2 hours
+                job = self.backend.with_delay().exchange_send(record)
+                with self.assertRaises(RetryableJobError):
+                    job.perform()
+            with freeze_time("2024-01-11 08:50:00"):
+                # + 23 hours and 50 minutes
+                job = self.backend.with_delay().exchange_send(record)
```

Comment:
Why not using the same job you've got before?
However, if the identity_key works well, here you should get still the same job but I think is useless to call send again.

Reply:
@SilvioC2C ping